docs/modules/ROOT/pages/how-tos/k3s_maintenance.adoc

```diff
@@ -0,0 +1,36 @@
+= K3S Maintenance
```

Comment:
As per https://products.docs.vshn.ch/products/appuio/managed/k3s.html we write `K3s` instead of `K3S`. Please change that in this document.

Reply:
done